### PR TITLE
Some Fixes, a Makefile and a Service 

### PR DESCRIPTION
**Pull request by Lerentis**
just a minor thing, but now the readme is correct again.
also: i could write a makefile as well as a systemd service for that if there is an interest in that

### CODE REVIEW

**Comment by Lerentis** [comment]
oh yeah could you enable issues for this repo again? i'm glad that there is someone willing to continue this project and i'd like to help as well

File: deskcon.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 
-BINPATH=`dirname $0`
-cd $BINPATH/server/
-./deskcon-server
\ No newline at end of file
+BINPATH=$(pwd)
+$BINPATH/server/deskcon-server.py
```
Comment (fbarriga) [inline deskcon.sh:4]: why we don't just do:
```
./server/deskcon-server.py
```
?

**Review by fbarriga — COMMENTED** [review]
@Lerentis Issues enabled. A `systemd service` and a `Makefile` would be nice to improve the distro packaging.

File: deskcon.sh (hunk 1 above)
Comment (Lerentis) [reply deskcon.sh:4]: was asking myself the same question. i just noticed that this script could have never worked as stated in the readme. there is no practical reason that this file is needed anyway. 
a better aproach would be to state in the documentation that `server/deskcon-server.py`should be called.

I will provide a systemd service for deskcon-server.py and a Makefile in the next few days when i get home

**Comment by Lerentis** [comment]
Added Makefile and Systemd Service for the Server

**Comment by fbarriga** [comment]
Cherry-picked and modified. Thanks!
Please check the new commits to see if everything is ok.

**Pull request closed without merging**